contrib/net/http/roundtripper.go

```diff
@@ -25,8 +25,11 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		tracer.Tag(ext.HTTPMethod, req.Method),
 		tracer.Tag(ext.HTTPURL, req.URL.Path),
 	}
-	if rate := rt.cfg.analyticsRate; rate > 0 {
-		opts = append(opts, tracer.Tag(ext.EventSampleRate, rate))
+	if rt.cfg.analyticsRate > 0 {
```

Comment:
Why did you change this? Please revert the change.

Reply:
For symmetry with the new if, as this is easier to read. But sure it's ok, I can revert the change.

Reply:
Thanks!